shared/src/metabase/mbql/schema.cljc

```diff
@@ -833,6 +856,14 @@
    (partial is-clause? :case)                        case
    :else                                             Field))
 
+#_((s/validator DatetimeExpression)
+   [:date-add [:date-add [:field 3 nil] 2 :year] 2 :month])
+
+
+#_((s/validator ArithmeticExpression)
+   [:+ [:+ [:field 3 nil] 2] 2])
+
```

Comment:
I'd personally rather not leave a bunch of comments for testing stuff all over the codebase. Take this and write an actual test

```suggestion
```

Reply:
Oh I forgot to delete these from debugging. Will move.